plugins/languages/nvim-orgmode.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "orgmode";
+  originalName = "nvim-orgmode"; 
+  defaultPackage = pkgs.vimPlugins.orgmode; 
+
+  maintainers = [ helpers.maintainers.GaspardMonge ]; 
+
+  settingsOptions = {
+    org_agenda_files = helpers.defaultNullOpts.mkStr "~/orgfiles/**/*" ''
```

Comment:
```suggestion
    org_agenda_files = lib.nixvim.defaultNullOpts.mkNullable with lib.types; either  str (listOf str) ""  ''
```

Reply:
<option type> <plugin default> <description> are the arguments with this library function. The upstream plugin accepts a string or list of strings, so we need to support that option type. It also doesn't have a default value upstream so we want to indicate that properly in the plugin default argument.